README.md

```diff
@@ -74,6 +74,10 @@ disable_pre_commit = false
 [tool.delfino.typecheck]
 strict_directories = ['src']
 
+[tool.delfino.plugins.test-unit]
+# See https://github.com/radeklat/delfino/blob/main/src/delfino/commands/pass_through_command.py for usage.
+pass_through_args = '--capture=no'
```

Comment:
This is a confusing naming because these are not actually "pass-through arguments". They are additional arguments, added to the default ones. Pass-through is something user specifies per execution and gets passed to the underlying tools. These are static. It will help reading the code as well. Because at the moment, `pass_through` refers to two things.

```suggestion
extra_args = '--capture=no'
```

Reply:
> They are additional arguments, added to the default ones.

What do you mean by "default ones"? Is it "default options/arguments for the underlying tools" or "default options/arguments for a click commands"?

If it is "default options/arguments for the underlying tools", I think the name, "pass_through_args", makes sense because they are options/arguments that are passed to the underlying tools.

If it is "default options/arguments for a click commands",  I think we should add one configuration per options/arguments. For example, when a click command definition is like [this one ](https://github.com/radeklat/delfino/pull/35/files#diff-8d935c69ae4824266e46c39894adfdfd79fbffb2a31ccf31a890c85467b02265R75), we should support config like the following. 
```
[tool.delfino.plugins.test-unit]
maxfail = 1
debug = True
pass_through_args = '--capture=no'
```

Instead of;

```
[tool.delfino.plugins.test-unit]
extra_args = '--maxfail 1 --debug -- --capture=no'
```

Because it is clear which options/arguments will be overwritten when configurations are used in combination with command line options in CLI. For example, when a user executes `delfino test-unit --maxfail=0`, it is clear that only `maxfail` configuration will be overwritten, with the former design.